code/modules/atmospherics/machinery/components/fusion/hfr_main_processes.dm

```diff
@@ -38,16 +38,14 @@
 	 */
 
 	critical_threshold_proximity_archived = critical_threshold_proximity
-	if(power_level >= 5)
-		critical_threshold_proximity = max(critical_threshold_proximity + max((round((internal_fusion.total_moles() * 1e5 + internal_fusion.temperature) / 1e5, 1) - 2500) / 200, 0), 0)
+	if(power_level == 6)
+		critical_threshold_proximity = max(critical_threshold_proximity + max((round((internal_fusion.total_moles() * 9e5 + internal_fusion.temperature) / 9e5, 1) - 2700) / 400, 0), 0)
 
-		critical_threshold_proximity = max(critical_threshold_proximity + max(log(10, internal_fusion.temperature) - 5, 0), 0)
-
-	if(internal_fusion.total_moles() < 1200 || power_level <= 4)
-		critical_threshold_proximity = max(critical_threshold_proximity + min((internal_fusion.total_moles() - 800) / 150, 0), 0)
+	if(internal_fusion.total_moles() < 2000 || power_level <= 5)
+		critical_threshold_proximity = max(critical_threshold_proximity + min((internal_fusion.total_moles() - 600) / 200, 0), 0)
 
 	if(internal_fusion.total_moles() > 0 && internal_fusion.temperature < 5e5 && power_level <= 4)
-		critical_threshold_proximity = max(critical_threshold_proximity + min(log(10, internal_fusion.temperature) - 5.5, 0), 0)
+		critical_threshold_proximity = max(critical_threshold_proximity + min(log(10, internal_fusion.temperature) - 5, 0), 0)
 
```

Comment:
> much easier to repair

Seems like the changes have instead made it much harder to repair:

- Previously the HFR needed to have the fusion mix go under 800 moles to have the first healing condition met, now it needs to go under under 600 moles.
- Previously the HFR needed to have the temperature drop below 5e5 Kelvin to have the second healing condition met, now it needs under 1e5 Kelvin.

The amount of healing possible also diminishes in both cases.

It's possible that these effects are dominated by the new clamping of  iron content to be between 0 and 1 (should damage taken from iron still be `max(round(iron_content, 1) - 1, 0)`, now that this is the case?), but I wanted to check to make sure. Running a small HFR fusion mix was the main way you could operate at dangerous fusion levels and make pre-Crystallizer Metal Hydrogen.

Reply:
the moles thing is a mistake on my part, it should increase to over 1200

the temperature drop is intentional

What i changed is making the damage less common, increased the temperature needed and moles, changed the iron content to be clamped (thanks for the tip about the damage, i'll change it)

currently i'm having thoughts about just removing the delamination part since i feel like is not that much engaging, but i don't know yet